### ISSUE TITLE
Add controller file to module folder

### ISSUE THREAD

**Issue by axixu** [ridwanskaterock/cicool#431]
Hi,

Is it possible, to add a new controller file to existing module?

I have tried and show me error 404 not found.

Thank you.

**Comment by ridwanskaterock** [comment]
Yes but you need add manually routes on application/config/routes.php

**Comment by ridwanskaterock** [comment]
https://codeigniter.com/userguide3/general/routing.html
